kzz7yh-f63981/cod-ve09v2_kzz7yh-f63981.xml

```diff
@@ -482,7 +486,7 @@
       <lb ed="#V" n="74"/>sed auget peccatum. 
    </p>
                <p xml:id="kzz7yh-f63981-d1e856">
-                  <lb ed="#V" n="75"/>Contra Galus 5 caro concupiscit aduersus spiritum spiritus 
+                  <lb ed="#V" n="75"/>Contra <ref>Galus 5</ref> caro concupiscit aduersus spiritum. spiritus 
```

Comment:
This is the abbreviations for "Galatinas" or "galatas" so we can just leave as Gal.
```suggestion
                  <lb ed="#V" n="75"/>Contra <ref>Gal. 5</ref> caro concupiscit aduersus spiritum. spiritus 
```